tsconfig.json

```diff
@@ -3,14 +3,14 @@
     "target": "es2021",
     "module": "commonjs",
     "declaration": true,
-    "outDir": "dist",
+    "outDir": "./dist",
     "strict": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "moduleResolution": "node",
     "experimentalDecorators": true,
-    "emitDecoratorMetadata": true
+    "emitDecoratorMetadata": true,
   },
-  "include": ["src"],
+  "include": ["src", "tests/helpers", "tests/helpers.ts"],
```

Comment:
Why did you add this?

Reply:
this is fake products for integration tests